realtime_trains/extract.py

```diff
@@ -1,33 +1,89 @@
-"""Gets the information from the realtime trains api"""
+"""Gets the information from the realtime trains API"""
```

Comment:
Slight nitpick, but if you renamed this from api to API, shouldn't you do the same across the whole file?

Reply:
It is API across the whole file except for variable/function names

Reply:
Guess we have to follow convention